check_powerdns_auth.py

```diff
@@ -18,34 +18,33 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ################################################################################
 
+from __future__ import division
+from __future__ import print_function
+from builtins import str
+from builtins import object
 __author__ = 'Remi Frenay <rf@worldstream.nl>'
-__version__ = '1.1'
+__version__ = '1.1.0'
```

Comment:
Current version is 1.1.0, should be bumped.

Reply:
Updated